apps/builder/app/builder/features/sidebar-left/panels/pages/settings.tsx

```diff
@@ -228,7 +227,7 @@ const FormFields = ({
   const faviconUrl = faviconAsset?.type === "image" ? faviconAsset.name : "";
 
   const project = projectStore.get();
-  const publishedUrl = new URL(getPublishedUrl(project?.domain ?? ""));
+  const publishedUrl = new URL(`https://${project?.domain}`);
```

Comment:
looks suspicious, wonder why its not needed any more

Reply:
Not sure it was needed at all. Internal logic of getPublishedUrl has `location` dependency, some BUILDER_URL env and other strange things